### PR TITLE
feat: release for Palette 3.3

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This is the release branch for 3.3

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1149#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/00-release-notes.md [hunk 1]
```diff
@@ -13,6 +13,39 @@ import InfoBox from 'shared/components/InfoBox';
...
+
+* CNI Packs:
+  * Calico CNI 3.25.0
+* CSI Packs:
+  * EBS CSI 1.16.0
+  * vSphere CSI 2.7.0
+* Add-on Packs:
+  * Flux v2 2.6.0
+  * Prometheus Operator 45.4.0
+  * MetalLB 0.13.9
```
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:47]:
```suggestion
  * MetalLB 0.13.9
  * Spectro Proxy 1.3.0
```

File: content/docs/00-release-notes.md [hunk 2]
```diff
@@ -13,6 +13,40 @@ import InfoBox from 'shared/components/InfoBox';
 import PointsOfInterest from 'shared/components/common/PointOfInterest';
 import Tooltip from "shared/components/ui/Tooltip";
 
+# March 17, 2023 - Release 3.3.0
```
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:16]:
```suggestion
# March 18, 2023 - Release 3.3.0
```

File: content/docs/00-release-notes.md [hunk 3]
```diff
@@ -13,6 +13,40 @@ import InfoBox from 'shared/components/InfoBox';
...
+## Edge
+
+### Features:
+
+* Edge now supports the ability to load images from an external OCI registry.
+* The Edge Installer can now include preloaded content bundles containing packages and artifacts. This is useful for scenarios where you work with limited internet bandwidth or want to optimize the installation process.
+* Users can create a custom Edge Installer that contains content bundles, operating systems, and user data.
+* Support for creating an Edge Installer ISO based on Virtual Machine Disks (VMDK) is now available.
+* Support for generating random UUID values for the Edge host is now available. This addresses the issue of some devices having the same Universal Unique Identifier (UUID) due to identical device identifiers.
+* The Palette Edge Host agent can now upgrade to newer versions automatically.
```
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:35]:
```suggestion

```

**Review by ritawatson — COMMENTED** [review]
LGTM

**Pull request merged**